src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final int MAX_VALUE = 10;
+    private static final int DEFAULT_FIGURE_VALUE = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
```

Comment:
a minor thing: please keep in mind that names of constants and variables should be as self-explanatory as possible.
It is very typical for programmers to write something and then have no clue what's going in the code when they return to it some time later and have to fix something.
I think it would be better to name those constants
"DEFAULT_SIZE" and "MAX_SIZE".